app/Models/BoardThread.php

```diff
@@ -51,19 +51,19 @@ class BoardThread extends Model
 
     protected $guarded = [];
 
-    public function cat(){
+    public function cat() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->